laletterbuilder/schema.py

```diff
@@ -25,3 +27,17 @@ def update_onboarding_info(cls, form, info: Dict[str, Any]):
     def perform_post_onboarding(cls, form, request: HttpRequest, user: JustfixUser):
         # TODO: Send SMS.
         pass
+
+
+@schema_registry.register_mutation
+class LaLetterBuilderChooseLetter(SessionFormMutation):
+    class Meta:
+        form_class = forms.ChooseLetterForm
+
+    login_required = False
+
+    @classmethod
+    def perform_mutate(cls, form, info: ResolveInfo):
+        # TODO: have this set a variable that decides which
+        # path the user goes down - perhaps just a redirect
+        return cls.mutation_success()
```

Comment:
why is the decided path something that needs to be saved server-side? Couldn't it just be a front-end link?